docs/manifest.json

```diff
@@ -246,16 +246,20 @@
       "implemented": ">=0.9.0",
       "contents": "storage/bucket.json"
     }, {
-      "title": "Channel",
-      "id": "channel",
-      "implemented": ">=0.26.0",
-      "contents": "storage/channel.json"
-    }, {
       "title": "File",
       "id": "file",
       "implemented": ">=0.9.0",
       "contents": "storage/file.json"
+    }, {
+      "title": "Channel",
+      "id": "channel",
+      "implemented": ">=0.26.0",
+      "contents": "storage/channel.json"
     }]
+  }, {
+    "title": "Translate",
+    "id": "translate",
+    "contents": "translate/index.json"
```

Comment:
You're going to want to add an implemented param, otherwise it will show up in every version.